projects/packages/post-list/src/index.js

```diff
@@ -0,0 +1,31 @@
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Closure function to copy the link to the clipboard.
+ *
+ * @return {Function} The click event handler.
+ */
+function copyLinkQuickAction() {
+	let timoutId;
+	/**
+	 * Copy the link to the clipboard.
+	 * @param {object} event - The event object.
+	 */
+	function onClick( event ) {
+		event.preventDefault();
+		clearTimeout( timoutId );
+		window.navigator.clipboard.writeText( event.target.getAttribute( 'href' ) ).then( () => {
+			event.target.textContent = __( 'Copied!', 'jetpack-post-list' );
+			timoutId = setTimeout( () => {
+				event.target.textContent = __( 'Copy link', 'jetpack-post-list' );
+			}, 2000 );
```

Comment:
Should we schedule a clearing of this timeout, just in case?

Reply:
Can you share a use case of what you have in mind and why we would need something more than this code?

Reply:
Imagine we're clicking the button to copy the link. Then, we navigate away before 2s have passed. This will cause a memory leak since the timeout won't be cleaned up. If it is a full page navigation, it should be fine, since the browser will have taken care of it (most likely), but what if it wasn't a full page nav? I think in the context of the admin this might be tricky to reproduce, since it doesn't have much possibilities for client-side navigation, but who knows? Maybe a plugin introduces some client-side nav in that view?